modules/update-mirror-repository.adoc

```diff
@@ -113,16 +125,3 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
```

Comment:
right after this, but before the `If the local container registry and the cluster are connected to the mirror host...` section (which we should keep below), is where [the *Creating the config map for image signature verification by using the oc CLI* content][1] steps fit in.

[1]: https://github.com/openshift/openshift-docs/blob/d4762f0f626a4dddb9d7330e63a3bb6cb73f5bb5/modules/update-oc-configmap-signature-verification.adoc